### ISSUE TITLE
aNother variaNt of rHack?

### ISSUE THREAD

**Issue by hasanalizxc** [Ranarrr/rN#4]
Hi there,

I realized this is "too" similar with rHack but couldn't see any information about this in README.

Can you explain this to us? And i saw a cfg named xHancer and as i know this is Turkish .cfg file. Is it your?

Thanks.

**Comment by Ranarrr** [comment]
When exactly did you "realize" that this hack is too similar to rHack? rHack is many years old, and I have not seen any source code from rHack. This is not rHack, and certainly not a similarity.

The .cfg file that is included in the binaries is a .cfg file I named because I reverse engineered a cheat named xHancer which I got from a friend, and added the way the creator of xHancer (yahowha) did the strafe helper. xHancer.cfg is just a copy of rN.cfg with some adjustments so that it uses strafe_control_helper 2 instead of 1. It's not a "Turkish .cfg file" whatever that means.
